### DISCUSSION TITLE
q-date date picker debounce?

### DISCUSSION THREAD

**Discussion by ssuess** [quasarframework/quasar#14011] [General - Components / Directives / etc]
Is there any easy way to make the q-date picker (which I am loading in a popup proxy) debounce a second or three before triggering v-model changes? Unlike other components there seems to be no debounce prop on it. Thanks!

**Comment by metalsadman** [comment]
there's a `debounce` function in the Quasar utils, you can use that in the `@update:model-value` handler.
